release.sh

```diff
@@ -1,29 +1,29 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
 OTP_RAW_VER=$(erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().'  -noshell)
-OTP_VER=$(echo ${OTP_RAW_VER} | sed -r 's/\"([0-9]+)\"/\1/g' | sed 's/[^0-9]*//g')
+OTP_VER=$(echo "${OTP_RAW_VER}" | sed -r 's/\"([0-9]+)\"/\1/g' | sed 's/[^0-9]*//g')
 
 yes_or_exit() {
     while true; do
         read -r -p "$1 " yn
         case $yn in
             [Yy][Ee][sS]) true; break ;;
-            [Nn][Oo]) echo "Ok, bye bye"; exit ;;
-            * ) echo "Please answer yes or no." ;;
+            [Nn][Oo]) echo "Ok, bye bye!"; exit ;;
+            * ) echo "Please answer 'yes' or 'no'." ;;
         esac
     done
 }
 
-if [ "${OTP_VER}" != "24" ]; then
-     echo "You are using Erlang/OTP ${OTP_VER_STR}"
-     echo "However, ex_doc must be built using OTP == 24 for compatability. Giving up..."
+if [ $((OTP_VER)) -lt 24 ]; then
```

Comment:
```suggestion
if [ $OTP_VER -ne 24 ]; then
```

Reply:
^ has to be exactly 24. If we publish with OTP 25 or 26, it will break for others.

Reply:
Ok.